### PR TITLE
[24_WofCard]

### LINKED ISSUE #24: Create WofCard component
Dumb component for wrapping more important content. 
Design based on Figma. It's just a box with secondary-color as background.
Should have a slot for real content. Should be flexible

### PR DESCRIPTION
**Pull request by karolkozlo**
closes #24 
[add] New WofCard component
[add] WofCard component is global component

### CODE REVIEW

File: frontend/src/components/WofCard.vue [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+@import './common.less';
+
+.card {
+    display: flex;
+    padding: 1em;
+    background-color: @secondary-color;
+    border-radius: 20px;
+    width: min-content;
+    height: min-content;
+}
```
Comment (kpagacz) [inline frontend/src/components/WofCard.vue:23]: Do we want this to be scoped?

**Review by kpagacz — APPROVED** [review]
My only gripe with this is it's so small xD
image

File: frontend/src/components/WofCard.vue (hunk 1 above)
Comment (karolkozlo) [reply frontend/src/components/WofCard.vue:23]: We will use BEM naming strategy and also this is kinda common, library component so here it's ok

File: frontend/src/components/WofCard.vue [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<template>
+    <div class="card">
+        <slot/>
```
[the remark this answers is not in the document]
Comment (karolkozlo) [reply frontend/src/components/WofCard.vue:3]: https://vuejs.org/v2/guide/components-slots.html

[the remark this answers is not in the document]
Comment (karolkozlo) [reply frontend/src/components/WofCard.vue:3]: This component is just another wrapper to get 'card style' look in our app. Size of cards depends on content

**Pull request merged**